Readme.md

```diff
@@ -4,24 +4,23 @@
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
 
-# Data Grid for Blazor - How to use DevExpress Reporting tools to implement export in a WASM application
+# Grid for Blazor - How to use DevExpress Reporting tools to implement export in a WebAssembly application
 
-The Data Grid was moved to maintenance support mode. No new features/capabilities will be added to this component. We recommend that you [migrate](https://docs.devexpress.com/Blazor/403162/grid/migrate-from-data-grid-to-grid) to the [Grid](https://docs.devexpress.com/Blazor/403143/grid) component and use its built-in [export functionality](https://demos.devexpress.com/blazor/Grid/Export/DataAwareExport).
+The [Grid](https://docs.devexpress.com/Blazor/403143/grid) component allows you to [export data](https://demos.devexpress.com/blazor/Grid/Export/DataAwareExport) to XLS, XLSX, and CSV file formats. You can also use DevExpress Reporting tools to implement export to different formats (PDF, XLSX, and DOCX). This example illustrates how to do this in a Blazor WebAssembly application.

```

Comment:
You referred to formats as targets for export operations. While you probably can say in this manner, I would still prefer referring to formats as circumstances because export operation targets are actual things - files, streams...

```suggestion
The [Grid](https://docs.devexpress.com/Blazor/403143/grid) component allows you to [export data](https://demos.devexpress.com/blazor/Grid/Export/DataAwareExport) in XLS, XLSX, and CSV file formats. You can also use DevExpress Reporting tools to implement export in different formats (PDF, XLSX, and DOCX). This example illustrates how to accomplish this task in a Blazor WebAssembly application.
```